sso/settings.py

```diff
@@ -59,7 +59,7 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
-CORS_ORIGIN_WHITELIST = ("http://localhost:3000",)
+CORS_ORIGIN_WHITELIST = ("http://localhost:3100",)
```

Comment:
I'm not sure this is going to work in prod from a distant web browser, but it's not crucial for the moment